src-electron/db/query-package.js

```diff
@@ -26,7 +26,7 @@ const dbEnum = require('../../src-shared/db-enum.js')
 const querySession = require('./query-session')
```

Comment:
Did anything change in this file? I might sendup having a big conflict on this from my PR.

Reply:
I think it's just a default formatter. We need to sync up with the formatter, so we all use the same. I didn't change any queries.